frontend/src/lib/components/TimelineSeekbar/TimelineSeekbar.tsx

```diff
@@ -23,25 +22,6 @@ export interface TimelineSeekbarProps {
     className?: string
 }
 
-const SEEKBAR_TOOLTIP_PLACEMENTS: Record<string, AlignType> = {
-    topRight: {
-        points: ['br', 'tr'],
-        offset: [7, 0], // To align with badges
-        overflow: {
-            adjustX: 0,
-            adjustY: 0,
-        },
-    },
-    topLeft: {
-        points: ['bl', 'tl'],
-        offset: [-7, 0], // To align with badges
-        overflow: {
-            adjustX: 0,
-            adjustY: 0,
-        },
-    },
-}
```

Comment:
This is a regression ([Storybook](https://61260360a8e859003a7c138b-thielmesix.chromatic.com/?path=/story/components-properties-timeline--multiple-points-for-one-person-property)):

### Master

<img width="281" alt="Screenshot 2024-02-13 at 15 41 33" src="https://github.com/PostHog/posthog/assets/4550621/0f5e1038-da28-4450-8c91-79d905588cf2">

### Here
<img width="289" alt="Screenshot 2024-02-13 at 15 41 16" src="https://github.com/PostHog/posthog/assets/4550621/217670fe-6920-4594-9b53-b7aff0e0e966">

Reply:
Fixed in https://github.com/PostHog/posthog/pull/20160/commits/95c185c25675bc573566a631febcc9d13ef36100

<img width="317" alt="Screenshot 2024-02-13 at 18 30 41" src="https://github.com/PostHog/posthog/assets/6685876/960584f4-40b1-4cce-a1bf-8ba656e43b63">